README.md

```diff
@@ -1,2 +1,72 @@
-# gridify
-A tool used to deploy projects using threefold grid 3 
+# Gridify
+
+A tool used to deploy projects using threefold grid 3.
+
+## Build
+
+Clone the repo and run the following command inside the repo directory:
+
+```bash
+make build
+```
+
+## Requirements
+
+- gridify uses [ginit](https://github.com/rawdaGastan/ginit) so `.Procfile` and `.env` must exist in root directory of your project
+- the project github repository must be public
+
+## Usage
```

Comment:
Also add a section about the configurations where they're saved on the filesystem.